rs/nns/governance/tests/governance.rs

```diff
@@ -13008,9 +12976,16 @@ async fn distribute_rewards_load_test() {
     );
 
     // Step 3.1: Inspect neurons to make sure they have been rewarded for voting.
-    for neuron in governance.neuron_store.heap_neurons().values() {
+    for id in neuron_range {
+        let (neuron, actual_maturity) = governance
+            .neuron_store
+            .with_neuron(&NeuronId { id }, |neuron| {
+                (neuron.clone(), neuron.maturity_e8s_equivalent)
+            })
+            .expect("Neuron not found");
+
         assert_ne!(
-            neuron.maturity_e8s_equivalent, maturity_e8s_equivalent,
+            actual_maturity, maturity_e8s_equivalent,
```

Comment:
I liked how it was before better.

Reply:
I changed the names to "current_neuron_maturity" and "starting_maturity" so that the assertion's clearer.